source/test/TestApp/PlayFabEventTest.h

```diff
@@ -57,11 +64,19 @@ namespace PlayFabUnit
             static int eventFailCount;
             static std::string eventFailLog;
 
+            static const unsigned int numThreads = 6;
+            static const unsigned int numEventsPerThread = 5;
+            static const unsigned int numTotalThreadedEvents = numThreads * numEventsPerThread;
+            static std::vector<std::thread> testThreadPool;
+            static unsigned int numEventsHeard;
```

Comment:
>numEventsHeard [](start = 32, length = 14)

nit: I'll leave it to your discretion but "heard" doesn't feel like a very precise way to describe what this value is measuring. "numEventsFired"? "numEventsSent"? "numEventsSentToPlaystream"?